### ISSUE TITLE
Reference of point cloud Laplacian

### ISSUE THREAD

**Issue by bearinsuke** [nmwsharp/robust-laplacians-py#13]
Hi,

Thanks for your wonderful contribution!
In your paper: [A Laplacian for Nonmanifold Triangle Meshes], you mentioned the method to build mesh Laplacian with cotangent weights. But I would like to ask about the reference paper in building your point cloud Laplacian.

Regards

**Comment by nmwsharp** [comment]
Hi! Thanks for the interest. You closed this so you may have already figured it out, but for posterity: 

The point cloud laplacian is described in Sec 5.7 of that same paper "A Laplacian for Nonmanifold Triangle Meshes". It works by applying the mesh approach to the union of a special collection of local neighborhood triangulations.

**Comment by bearinsuke** [comment]
Hi! Yeah, I found the corresponding part in your paper. But I still have a small question: Generally, do you still hold the common way of building Point Cloud Laplacian, with steps of (1) k-nearest neighbors, (2) tangent plane, (3) Delaunay triangulation？ I am also not sure that: your major contribution here is to use a tufted cover to optimize the triangulation, but the skeleton of Laplacian computation still follows the previous 3 steps, right?
Thanks!

**Comment by nmwsharp** [comment]
Yep, you've got it right. There are a bunch of subtleties in the details of the various methods, but that is the right structure. And then the Tufted Cover gets applied to the triangulation at the end.

There is also some discussion of this in the second paragraph of Sec 5.7 of the paper, putting the approach in context with others which follow the same strategy.

**Comment by bearinsuke** [comment]
Thank you very much for replying this quickly! Besides, I am also quite curious about the volume defined in point cloud Laplacian. Is that equal to the Voronoi area corresponding to the point in the local mesh (Delaunay)? How do we compute that?

**Comment by bearinsuke** [comment]
Hi,
I am also quite interested in what your mass matrix in point cloud Laplacian means. Could you please give some hints?
